src/MeetingsSDKAdapter/controls/AudioControl.js

```diff
@@ -0,0 +1,62 @@
+import {Observable} from 'rxjs';
+import {map, distinctUntilChanged} from 'rxjs/operators';
+import {MeetingControlState} from '@webex/component-adapter-interfaces';
+import MeetingControl from './MeetingControl';
+
+/**
+ * Display options of a meeting control.
+ *
+ * @external MeetingControlDisplay
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L58}
+ */
+
+export default class AudioControl extends MeetingControl {
+  /**
+   * Attempts to mute the microphone of the given meeting ID.
+   * If the microphone is successfully muted, an audio mute event is dispatched.
+   *
+   * @private
+   * @param {string} meetingID ID of the meeting to mute audio
+   */
+  action(meetingID) {
```

Comment:
to be able to test this function, it should return the promise that handleLocalAudio returns. This way, we can `await` for the `action()` in the tests before making assertions

Reply:
```js
return this.adapter.handleLocalAudio(...);
```
or, to the same effect
```js
await this.adapter.handleLocalAudio(...);
```